vpr/src/route/netlist_routers.h

```diff
@@ -37,6 +38,8 @@ struct RouteIterResults {
     bool is_routable = true;
     /** Net IDs with changed routing */
     std::vector<ParentNetId> rerouted_nets;
+    /** Net IDs with changed bounding box */
```

Comment:
IDs of nets whose bounding box was expanded by the current routing iteration. Necessary to determine when partition trees should be rebuilt during parallel routing.